code-challenges.test.js

```diff
@@ -10,47 +13,206 @@
+//     })
+//   })
 
 // --------------------1) Create a function that takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence.
 
 // a) Create a test with expect statements for each of the variables provided.
 
+describe("fibonacciSequence" , () => {
+    
 const fibLength1 = 6
 // Expected output: [1, 1, 2, 3, 5, 8]
 
 const fibLength2 = 10
 // Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
 
+    it("takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence", () => {
+      expect(fibonacciSequence(fibLength1)).toEqual([1, 1, 2, 3, 5, 8])
+      expect(fibonacciSequence(fibLength2)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
+    })
+  })
+
+
+
+
+
+
 // b) Create the function that makes the test pass.
 
 
+// Psuedo Code
+// Looking what the fibonacci sequence is. From my research the value in the array is created by the sum of the first two indexs
+// create a function called fibonacciSequence
+//  im going to need two indexes to start the fib sequence which would be 1,1
+// i would need to use a method to get the sum of these two numbers
+//  i would use .map to iterate throughout the indexs 
+// input:taking number > 2
+// output:return of an array that length containing the numbers of the Fibonacci sequence
+
+
+// const fibonacciSequence = (n) => {
+//      const arr = [1,1];
+//      let a,b;
+//      for (let i = 0; i < number; i++) {
+//          a= arr[i - 1];
+//          b = arr[i - 2];
+//          arr.push(a + b);
+//      }
+//      return arr;
+//      }
+//      console.log(fibonacciSequence(6))
+//      console.log(fibonacciSequence(10));
+
+    //RECIEVED FOR OUTPUT:  FAIL  ./code-challenges.test.js
+    //  fibonacciSequence
+    //    ✕ takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence (8 ms)
+   
+    //  ● fibonacciSequence › takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence
+   
+    //    expect(received).toEqual(expected) // deep equality
+   
+    //    - Expected  - 4
+    //    + Received  + 0
+   
+    //      Array [
+    //        1,
+    //        1,
+    //    -   2,
+    //    -   3,
+    //    -   5,
+    //    -   8,
+    //      ]
+   
+    //      36 |
+    //      37 |     it("takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence", () => {
+    //    > 38 |       expect(fibonacciSequence(fibLength1)).toEqual([1, 1, 2, 3, 5, 8])
+    //         |                                             ^
+    //      39 |       expect(fibonacciSequence(fibLength2)).toEqual([1, 1, 2, 3, 5, 8, 13, 21, 34, 55])
+    //      40 |     })
+    //      41 |   })
+   
+    //      at Object.toEqual (code-challenges.test.js:38:45)
+
+
+// Where i got stuck is I couldnt get FibLength two to log the output
```

Comment:
Ah I see you have 'n' as a parameter and not 'number'. Other than that it should work!